src/app/main.cpp

```diff
@@ -132,6 +135,9 @@ void usage( const QString &appName )
       << QStringLiteral( "\t[--dxf-scale-denom scale]\tscale for dxf output\n" )
       << QStringLiteral( "\t[--dxf-encoding encoding]\tencoding to use for dxf output\n" )
       << QStringLiteral( "\t[--dxf-preset maptheme]\tmap theme to use for dxf output\n" )
+      << QStringLiteral( "\t[--config-local-storage]\tstore user config folder in local machine folder only.\n" )
+      << QStringLiteral( "\t[--config name]\tLoad a named config from the users config folder.\n" )
+      << QStringLiteral( "\t[--config-storage-location path]\tPath to store user config folders..\n" )
```

Comment:
Looks like the description of the other options don't start with caps, would be good to align.
Also, I think it would be worth translating the description. Or are there technical reasons in place that don't allow this?